### PR TITLE
Issue #43 Rss Feed

### LINKED ISSUE #43: RSS/Atom feed
https://www.24life.com/feed/

https://github.com/hlxsites/vg-macktrucks-com/blob/main/.github/workflows/generate-atom-news-feeds.yaml

The generated file will be named feed.xml, so we also need to add a redirect from /feed/ to /feed.xml

### PR DESCRIPTION
**Pull request by iamitshri**
Fix #43

Test URLs:
- Before: https://main--24life--hlxsites.hlx.page/
- After: https://43-rssatom-feed--24life--hlxsites.hlx.page

### CODE REVIEW

File: .github/ci/generate-news-feed.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import fs from 'fs';
+
+/**
+ * @type {FeedConfig[]}
+ */
+const feeds = [
+  {
+    title: '24life Updates',
+    targetFile: `../../feed.xml`,
+    source: 'https://main--24life--hlxsites.hlx.live/feed.json',
```
Comment (iamitshri) [inline .github/ci/generate-news-feed.js:11]: @synox, I'm not sure what the source should be?  should it be https://www.24life.com/feed/?

File: .github/ci/generate-news-feed.js [hunk 2]
```diff
@@ -8,7 +9,7 @@ const feeds = [
   {
     title: '24life Updates',
     targetFile: `../../feed.xml`,
-    source: 'https://main--24life--hlxsites.hlx.live/feed.json',  
+    source: 'https://main--24life--hlxsites.hlx.page/articles.json',
```
Comment (synox) [inline .github/ci/generate-news-feed.js]: use .live, we only want published articles here

File: .github/ci/generate-news-feed.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+/**
+ * @type {FeedConfig[]}
+ */
+const feeds = [
+  {
+    title: '24life Updates',
+    targetFile: `../../feed.xml`,
+    source: 'https://main--24life--hlxsites.hlx.live/articles.json',
+    siteRoot: "https://main--24life--hlxsites.hlx.live",
```
Comment (synox) [inline .github/ci/generate-news-feed.js:12]: Let's point to the url that will be post go-live: https://www.24life.com/

File: .github/ci/generate-news-feed.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    link:	"https://main--24life--hlxsites.hlx.live",
```
Comment (synox) [inline .github/ci/generate-news-feed.js:13]: In macktruck this links to this feed itself. check the rss/atom specs if that is the correct usage.  e.g. `https://www.24life.com/feed.xml`

File: .github/ci/generate-news-feed.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
+import { Feed } from 'feed';
+import fs from 'fs';
+
+/**
+ * @type {FeedConfig[]}
+ */
+const feeds = [
+  {
+    title: '24life Updates',
```
Comment (synox) [inline .github/ci/generate-news-feed.js:9]: The original feed just says `24Life`

File: .github/ci/generate-news-feed.js (hunk 4 above)
Comment (iamitshri) [reply .github/ci/generate-news-feed.js:13]: Thanks !  I dint know 
https://validator.w3.org/feed/docs/atom.html#requiredFeedElements:~:text=A%20feed%20should%20contain%20a%20link%20back%20to%20the%20feed%20itself

**Review by synox — APPROVED** [review]

**Pull request merged**